source/contracts/reporting/Market.sol

```diff
@@ -680,4 +681,12 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         }
         return _winningDestination.getUniverse().getParentPayoutDistributionHash();
     }
+
+    // Markets hold the initial fees paid by the creator in ETH and REP, so we dissallow ETH and REP extraction by the controller
```

Comment:
Markets also hold all of the order escrow, which means the denomination token and all share tokens for this market.  This logic lives over in `Order.sol`.

Reply:
It also escrows denomination token when complete sets are purchased in `CompleteSet.sol`.

Reply:
This is still missing all of the share tokens for the market, which are held in escrow by Market for open orders.  this means we'll need `i=3` through up to `i=8` for `_protectedTokens[i]`.